crates/transport_tx5/src/test.rs

```diff
@@ -371,9 +374,10 @@ async fn nonexistent_peer_marked_unresponsive() {
         tx_handler1.clone(),
     );
 
-    let faulty_url = Url::from_str(
-        "ws://127.0.0.1:40813/VtK2IOCncQM6LbWkvhB_CYwajQzw6Dii-Oc-0IRtHmc",
-    )
+    let faulty_url = Url::from_str(format!(
```

Comment:
If we don't use the right signal URL then yes you get an error back but it never attempts to make a connection because it fails to set up the signal connection. Not really what the test is supposed to be checking.

Reply:
Hmm...yes and slightly yes. The code that marks peer unresponsive also doesn't distinguish these two cases, right? So the test was testing as far as the actual code can distinguish but not what the code may ideally want to be able to distinguish. But I agree, it's certainly better with a valid signal url.

Reply:
Yeah that's fair, the test as it was tested that any error would result in a peer getting marked unreachable. It was fine in that respect. It wasn't useful to me with checking the effect on the connection status without this change.